panel/models/quill.ts

```diff
@@ -31,8 +31,8 @@ export class QuillInputView extends PanelHTMLBoxView {
       this.quill.enable(!this.model.disabled)
       this._editing = false
     })
-    const {mode, toolbar} = this.model.properties
-    this.on_change([mode, toolbar], () => {
+    const {mode, toolbar, placeholder} = this.model.properties
+    this.on_change([mode, toolbar, placeholder], () => {
```

Comment:
A full rerender kind of seems like overkill for a placeholder change. While Quill.js doesn't provide an official API for updating the placeholder this works:

```suggestion
    this.on_change([placeholder], () => {
      this.quill.root.setAttribute('data-placeholder', this.model.placeholder)
    })
    this.on_change([mode, toolbar], () => {
```

Reply:
You are completely right.